docs/src/stake-pool.md

```diff
@@ -1,11 +1,427 @@
+Stake activates starting from epoch: 161
+Delegated Vote Account Address: 2HUKQz7W2nXZSwrdX5RkfS2rLU4j1QZLjdGCHcoUKFh3
+Stake Authority: 4SnSuUtJGKvk2GYpBwmEsWG53zTurVM8yXGsoiZQyMJn
+Withdraw Authority: 4SnSuUtJGKvk2GYpBwmEsWG53zTurVM8yXGsoiZQyMJn
+```
+
+### Example: List validator stake accounts
+
+In order to deposit into the stake pool, a user must first delegate some stake
+to one of the validator stake accounts associated with the stake pool. The
+command-line utility has a special instruction for finding out which vote
+accounts are already associated with the stake pool.
+
+```sh
+$ spl-stake-pool list --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC
+CrStLEWfme37kDc3nubK9HsmWR5dsuVUuqEKqTR4Mc5E    1.002282880 SOL
+E5KBATUd21Dnjnh5sGFw5ngp9kdVXCcAAYMRe2WsVXie    1.002282880 SOL
+FYQB64aEzSmECvnG8RVvdAXBxRnzrLvcA3R22aGH2hUN    1.002282880 SOL
+FhFft7ArhZZkh6q4ir1JZMYFgXdH6wkT5M5nmDDb1Q13    1.002282880 SOL
+Total: 4.009131520 SOL
+```
+
+If the manager has recently created the stake pool, and there are no stake
+accounts present yet, the command-line utility will inform us.
+
+```sh
+$ spl-stake-pool list --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC
+No accounts found.
+```
+
+### Example: Deposit stake
+
+Stake pools only accept deposits from fully staked accounts, so we must first
+create stake accounts and delegate them to one of the validators managed by the
+stake pool. Using the `list` command from the previous section, we see that
+`2HUKQz7W2nXZSwrdX5RkfS2rLU4j1QZLjdGCHcoUKFh3` is a valid vote account, so let's
+create a stake account and delegate our stake there.
+
+```sh
+$ solana-keygen new --no-passphrase -o stake-account.json
+Generating a new keypair
+Wrote new keypair to stake-account.json
+============================================================================
+pubkey: 4F4AYKZbNtDnu7uQey2Vkz9VgkVtLE6XWLezYjc9yxZa
+============================================================================
+Save this seed phrase to recover your new keypair:
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+============================================================================
+$ solana create-stake-account stake-account.json 10
+Signature: 5Y9r6MNoqJzVX8TWryAJbdp8i2DvintfxbYWoY6VcLEPgphK2tdydhtJTd3o3dF7QdM2Pg8sBFDZuyNcMag3nPvj
+$ solana delegate-stake 4F4AYKZbNtDnu7uQey2Vkz9VgkVtLE6XWLezYjc9yxZa 2HUKQz7W2nXZSwrdX5RkfS2rLU4j1QZLjdGCHcoUKFh3
+Signature: 2cDjHXSHjuadGQf1NQpPi43A8R19aCifsY16yTcictKPHcSAXN5TvXZ58nDJwkYs12tuZfTh5WVgAMSvptfrKdPP
+```
+
+Two epochs later, when the stake is fully active and has received one epoch of
+rewards, we can deposit the stake into the stake pool.
+
+```sh
+$ spl-stake-pool deposit --pool 3CLwo9CntMi4D1enHEFBe3pRJQzGJBCAYe66xFuEbmhC --stake 4F4AYKZbNtDnu7uQey2Vkz9VgkVtLE6XWLezYjc9yxZa
+Depositing into stake account FYQB64aEzSmECvnG8RVvdAXBxRnzrLvcA3R22aGH2hUN
+Creating account to receive tokens 34XMHa3JUPv46ftU4dGHvemZ9oKVjnciRePYMcX3rjEF
```

Comment:
> Creating account to receive tokens 34XMHa3JUPv46ftU4dGHvemZ9oKVjnciRePYMcX3rjEF

Can the user optionally pass in the token address?  Having to parse the cli output to determine the address will be hard to use from a script

Reply:
Yep, you can use the `token-receiver` param.  I'll add an example for that